packages/storybook-react/src/stories/ButtonLink/argTypes.ts

```diff
@@ -0,0 +1,10 @@
+export const argTypes = {
+  appearance: {
+    type: 'select',
+    options: ['primary-action-button', 'secondary-action-button', 'subtle-button'],
```

Comment:
Both the CSS module and the React module for ButtonLink don't yet support `appearance`. We should spend some time in the future to implement this, but until that time we shouldn't have it in Storybook.